test/k8sT/Services.go

```diff
@@ -1113,7 +1113,73 @@ var _ = Describe("K8sServicesTest", func() {
 			testExternalTrafficPolicyLocal()
 		})
 
-		SkipContextIf(helpers.RunsWithoutKubeProxy, "with L4 policy", func() {
+		Context("TFTP with DNS Proxy port collision", func() {
+			var (
+				demoPolicy    string
+				ciliumPodK8s1 string
+				ciliumPodK8s2 string
+				DNSProxyPort1 int
+				DNSProxyPort2 int
+			)
+
+			BeforeAll(func() {
+				var err error
+				ciliumPodK8s1, err = kubectl.GetCiliumPodOnNodeWithLabel(helpers.CiliumNamespace, helpers.K8s1)
+				Expect(err).Should(BeNil(), fmt.Sprintf("Cannot get cilium pod on %s", helpers.K8s1))
```

Comment:
nit: you don't have to use `fmt.Sprintf`, the most of gomega functions (including `Should`) already support formatting.

```suggestion
				Expect(err).Should(BeNil(), "Cannot get cilium pod on %s", helpers.K8s1)
```

Reply:
Fixed